File: tests/fixtures/csharp/basic/Calculator.cs
using System;

namespace CalculatorApp
{
    public class Calculator
    {
        public int Add(int a, int b)
        {
            return a + b;
        }

        public int Multiply(int a, int b)
        {
            return a * b;
        }
    }

    public class Program
    {
        public static void Main(string[] args)
        {
            var calc = new Calculator();
            int sum = calc.Add(1, 2);
            int product = calc.Multiply(sum, 3);
            Console.WriteLine($"Result: {product}");
        }
    }
}

File: tests/fixtures/php/basic/Calculator.php
<?php

class Calculator
{
    public function add(int $a, int $b): int
    {
        return $a + $b;
    }

    public function multiply(int $a, int $b): int
    {
        return $a * $b;
    }
}

$calc = new Calculator();
echo $calc->add(1, 2) . "\n";
echo $calc->multiply(3, 4) . "\n";
